common/sai.py

```diff
@@ -154,12 +157,15 @@ def uint32(self):
         return int(self.value())
 
 
-class Sai:
+class Sai(AbstractEntity):
```

Comment:
Preferably use composition over inheritance

Reply:
Here we have inheritance everywhere.
IMO it depends on the use case. Could you please explain why it is better in this particular case?